legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java

```diff
@@ -13,10 +13,12 @@
 	private Long id;
 	private LegacyArrowheadService providedService;
 	private LegacyArrowheadSystem provider;
-	private String serviceUri;
+	private String serviceURI;
 	private Boolean udp;
 	private LocalDateTime endOfValidity;
 	private Integer version = 1;
+	private Integer ttl = 0;
+	private String metadata;
```

Comment:
What is the reason of these fileds' necessity despite the fact they are not used at all? If your application is sending such fileds, then it's not a problem from the translator. They will be simply ignored

Reply:
The problem I had is not at sending but at receiving. When I get the response, the data format without these fields is checked as a wrong format, so an error is triggered.
This is mainly to maintain the old interface

Reply:
Okay understand, we can have these fileds then